tutorial/extending-widgets.md

```diff
@@ -68,3 +68,91 @@ func main() {
 	w.ShowAndRun()
 }
 ```
+
+We can also extend a widget and its renderer, allowing you to override its 
```

Comment:
The "and its" here does not seem needed - really we are only covering extending a renderer in this section